test/integration/dialog-form-layout-test-styles.js

```diff
@@ -0,0 +1,11 @@
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+// TODO: this should be eventually covered by base styles
+registerStyles(
+  'vaadin-dialog-overlay',
+  css`
+    [part='content'] {
+      padding: 1.5rem;
```

Comment:
Some form-layout breakpoints were relying on the dialog padding so I had to add it for tests to pass.